src/design-system/stylesheets/components/_tabs.scss

```diff
@@ -91,6 +91,8 @@
 }
 
 .app-tabs__container pre {
+  max-width: inherit;
```

Comment:
Is there a way we can avoid undoing CSS?

Reply:
Cascade
tabs `pre` styling comes before `pre`

Reply:
I think we should make the selector more specific, overriding CSS is an anti-pattern.

Reply:
The only thing we could try would be to use direct descendant selector within the prose scope? But that might not work in all circumstances.

The `<pre>` we're trying to target doesn't have any class names or anything else we can hook onto as it's come from the generated markdown.

Reply:
it's unfortunate that both are in `.app-prose-scope`

Reply:
I think the direct descendant would work for this case.

If not it's fairly easy to add classes to the output: https://github.com/alphagov/govuk-design-system/compare/markdown-heading-ids-linkable

Reply:
Given this is an existing issue (we already have CSS in the example component that is there only to 'undo' styling from the prose component) can we raise an issue about this and not block this PR?

Reply:
Sure, I wasn't aware we were already doing this approach, sorry both.